service-providers/dotmatrix/25-Q2-Marketing-Ecosystem-Report.md

```diff
@@ -0,0 +1,54 @@
+---
+Applicant: DotMatrix LLP.
+Type: Ecosystem & Community Growth
+Timeline: 2025-Q2
+Investment: 7,500 USDC (2,500 USDC/month)
+Payment: 10% in RING (monthly average price) + 90% in USDC (0xfurqan.eth on Arbitrum)
```

Comment:
```suggestion
Payment: 10% in RING (monthly average price) + 90% in USDC (0xfurqan.eth on Base)
```

Can we change to Base here, thanks!

Please make sure your address can accept payment in USDC on Base chain.